packages/file/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: file
-version: 6.2.0
+version: 7.0.0
```

Comment:
We can set the version to `7.0.0-wip` to indicate this isn't  yet ready to publish (until the other PR also lands). Same for the changelog. Then we could probably merge this any time.